From: =?utf-8?b?IlNaIExpbiAo5p6X5LiK5pm6KSI=?= <szlin@debian.org>
Date: Fri, 28 Sep 2018 16:30:11 +0800
Subject: Fix FTBFS with bindings/cxx/examples/gpiod_cxx_tests.cpp

This patch adds proper include file to fix FTBFS issue in gpiod_cxx_tests.cpp,
please see below error.

gpiod_cxx_tests.cpp:29:19: note: 'std::function' is defined in header '<functional>'; did you forget to '#include <functional>'?
gpiod_cxx_tests.cpp:26:1:
+#include <functional>
---
 bindings/cxx/examples/gpiod_cxx_tests.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/bindings/cxx/examples/gpiod_cxx_tests.cpp b/bindings/cxx/examples/gpiod_cxx_tests.cpp
index df4820b..767fe98 100644
--- a/bindings/cxx/examples/gpiod_cxx_tests.cpp
+++ b/bindings/cxx/examples/gpiod_cxx_tests.cpp
@@ -21,6 +21,7 @@
 #include <map>
 #include <cstring>
 #include <cerrno>
+#include <functional>
 
 #include <poll.h>
 
